EventListener/UserContextSubscriber.php

```diff
@@ -123,22 +130,25 @@ public function onKernelResponse(FilterResponseEvent $event)
             return;
         }
 
-        $response = $event->getResponse();
-        $vary = $response->getVary();
+        if($this->alwaysVary)
+        {
```

Comment:
code style: the { should go on the same line as the if, with spaces like this: `if ($this->addVaryOnHash) {`